### DISCUSSION TITLE
Where to use this restrict client access step in authentication flow when I have linked Keycloak with Azure AD?

### DISCUSSION THREAD

**Discussion by Sangamesh21** [sventorben/keycloak-restrict-client-auth#204] [General]
Hi Sven,

I am working with Keycloak. Currently, I have linked my keycloak with Azure AD and it works fine. I used first broker login as first login. There is an internal application at my company where I have to do SSO configuration via keycloak and Azure AD. That also works well. But currently I need to restrict client application access to few users or group of users in Keycloak. I tried out by creating  Client group policy. but I could still login with users who aren't in group too. Then I came across this extension. I wanted to know where  do I need to add this restrict client auth step in the authentication flow. I did try following your documentation and video but it didn't work for me for identity provider. Could you please let me know where to add it? 

Thanks

**Comment by sventorben** [comment]
Hey @Sangamesh21,

I did not find the time to update the documentation (see. #117), yet.
Do you still need help with this?

Regards
Sven-Torben

**Comment by Sangamesh21** [comment]
Hey @sventorben,

Thank you. It did work for me. Good work indeed.
Appreciate it !!

Thanks

Regards
Sangamesh
